bubblesub/tests/test_command_naming.py

```diff
@@ -112,6 +112,6 @@ def test_commands_have_names(  # pylint: disable=redefined-outer-name
     assert len(classes) >= 1
     for cls in classes:
         assert isinstance(cls.names, list)
-        assert len(cls.names) > 0
+        assert bool(cls.names)
```

Comment:
This obscures the purpose of the assertion

Reply:
True, disable warning approach